### PR TITLE
lychee: add page

### PR DESCRIPTION
**Pull request by Managor**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by Managor** [comment]
I was reminded if this tool by this comment https://github.com/tldr-pages/tldr/pull/13711#issuecomment-2355812117. Gonna run this project through this tool and add more commands as I use it

**Comment by Managor** [comment]
It was as easy as running `grep -r "More information:" | lychee -`. Not sure how I should format that in the tldr page.

**Comment by make17better** [comment]
🤩 Amazing tool, thank you for sharing lychee.

File: pages/common/lychee.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+> Find broken URLs.
+> More information: <https://lychee.cli.rs/usage/cli/>.
+
+- Scan a website for broken links:
+
+`lychee {{https://example.com}}`
+
+- Display breakdown of error types:
+
+`lychee --form detailed {{https://example.com}}`
```
Comment (acuteenvy) [inline pages/common/lychee.md:12]: `--form` is not an option.

> --format <FORMAT>
          Output format of final status report (compact, detailed, json, markdown) [default: compact]


```suggestion
- Display a breakdown of error types:

`lychee --format detailed {{https://example.com}}`
```

**Review by acuteenvy — APPROVED** [review]

**Pull request merged**